tool/build-robots-txt.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * This script generates a /robots.txt file that depends on
+ * process.env.BUILD_ALWAYS_ALLOW_ROBOTS.
+ *
+ */
+const fs = require("fs");
+
+const { ALWAYS_ALLOW_ROBOTS } = require("../build/constants");
+
+const ALLOW_TEXT = `
+User-agent: *
+
+Disallow: /api/
```

Comment:
@peterbe I'm concerned about two things:
- The missing `Sitemap` directive. Does Google use the sitemap location configured in our search console? I couldn't find proof of that.
- I really don't think we should index our localized search pages (`/en-US/search`, `fr/search`, etc.). It seems that the prevailing opinion is that one shouldn't allow Google to index them because in our case they could distract from the actual document pages. Also, my understanding is that indexing these useless pages burns our crawl budget.

Reply:
I side with @escattone here.
The argument about the inline meta element is nice.
But a crawler (say, Bing), will only learn about it once it parses the file.

A robots.txt entry could make it not query the page in the first place.

Reply:
Alrite. I won't be weird and try to introduce change :)